src/locale/locales/fr/messages.po

```diff
@@ -2130,13 +2127,9 @@ msgstr "Fil d’actu"
 msgid "Feed by {0}"
 msgstr "Fil d’actu par {0}"
 
-#: src/view/screens/Feeds.tsx:709
-#~ msgid "Feed offline"
-#~ msgstr "Fil d’actu hors ligne"
-
 #: src/components/StarterPack/Wizard/WizardListCard.tsx:52
 msgid "Feed toggle"
-msgstr ""
+msgstr "Bouton pour le fil d’actu"
```

Comment:
I don't have a good idea here (as I explained in the standalone comment before this review). This suggestion try to workaround the problem by ignoring the English label and trying to find a text I would write on such a toggle.

```suggestion
msgstr "Ajouter/enlever le fil d’actu"
```